operator/cmd/mesh/install.go

```diff
@@ -345,25 +346,19 @@ func detectIstioVersionDiff(p Printer, tag string, ns string, kubeClient kube.CL
 			}
 		}
 		revision := manifest.GetValueForSetFlag(setFlags, "revision")
-		msg := ""
-		// when the revision is not passed and if the ns has a prior istio
-		if revision == "" && tag != icpTag {
-			if icpTag < tag {
-				msg = "A newer"
-			} else {
-				msg = "An older"
-			}
-			p.Printf("%s Istio control planes installed: %s.\n"+
-				"%s "+msg+" installed version of Istio has been detected. Running this command will overwrite it.\n", warnMarker, strings.Join(icpTags, ", "), warnMarker)
-		}
 		// when the revision is passed
-		if icpTag != "" && tag != icpTag && revision != "" {
+		if icpTag != "" && tag != icpTag {
+			check := "         Before upgrading, you may wish to use 'istioctl x precheck' to check for upgrade warnings.\n"
```

Comment:
nit: I think it's better to caluclate the padding using the warnMarker length, but likely It won't change at all.